src/problem_bank_scripts/problem_bank_scripts.py

```diff
@@ -329,29 +329,21 @@ def read_md_problem(filepath):
             block_count += 1
 
             if block_count == 1:
-                blocks["block{0}".format(block_count)] = [
-                    x,
-                ]
+                blocks[f"block{block_count}"] = [x]
             else:
-                blocks["block{0}".format(block_count - 1)].append(x)
-                blocks["block{0}".format(block_count)] = [
-                    x,
-                ]
+                blocks[f"block{block_count - 1}"].append(x)
+                blocks[f"block{block_count}"] = [x]
```

Comment:
I missed that my autoformatter/linter in vscode for other projects changed some things here ☹️  - I can revert if needed